api/api.py

```diff
@@ -0,0 +1,109 @@
+import pickle
+from typing import List
+from .exceptions import InvalidModelError
+
+from sentence_transformers import SentenceTransformer
+
+
+class Ner4Opt(object):
+
+    CLASSICAL = 'classical'
+    CLASSICAL_PLUS = 'classical_plus'
+    DEEP = 'deep'
+    HYBRID = 'hybrid'
+
+    def __init__(self, model_name: str = 'hybrid'):
+        """Init Model."""
+
+        self._model_name = model_name
+
+        if model_name == Ner4Opt.CLASSICAL:
+            self._model = pickle.load(open('models/classical.pkl', 'rb'))
+
+        elif model_name == Ner4Opt.CLASSICAL_PLUS:
+            self._model = pickle.load(open('models/classical_plus.pkl', 'rb'))
+
+        elif model_name == Ner4Opt.DEEP:
+            self._model = SentenceTransformer("skadio/optimization_finetuned_roberta")
+
+        elif model_name == Ner4Opt.HYBRID:
+            self._model = pickle.load(open('models/hybrid_crf_model.pkl', 'rb'))
```

Comment:
curious how they hbyrid will access the SentenceTransfomer from opt_tuned_roberta above

Reply:
Great point, I will make sure there wouldn't be any duplicate instantiations.